includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -465,6 +464,33 @@ protected function get_row_data( $item ) {
 					$user_average_grade = Sensei_Utils::quotient_as_absolute_rounded_number( $grade_total, $grade_count, 2 );
 				}
 
+				// Get the last lesson activity.
+				$last_activity_comment = Sensei_Utils::sensei_check_for_activity(
+					array(
+						'number'  => 1,
+						'user_id' => $item->ID,
+						'type'    => 'sensei_lesson_status',
+					),
+					true
+				);
+
+				$last_activity = null;
+				if ( $last_activity_comment ) {
+					if ( $this->csv_output ) {
+						$last_activity = $last_activity_comment->comment_date;
+					} else {
+						$last_activity = sprintf(
+							'<abbr title="%s">%s</abbr>',
+							$last_activity_comment->comment_date,
+							sprintf(
+								/* translators: Time difference between two dates. %s: Number of seconds/minutes/etc. */
+								esc_html__( '%s ago', 'sensei-lms' ),
+								human_time_diff( strtotime( $last_activity_comment->comment_date_gmt ) )
```

Comment:
Could we show the full date after 6 days of inactivity, and use `human_time_diff` otherwise?